wp-includes/service-workers.php

```diff
@@ -225,7 +225,7 @@ function wp_disable_script_concatenation() {
 	 * for browsers that don't support service workers. Note that concatenation only applies in the admin,
 	 * for authenticated users without full-page caching.
 	*/
-	if ( isset( $_COOKIE['wordpress_sw_installed'] ) ) {
+	if ( isset( $_COOKIE['wordpress_sw_installed'] ) ) {  // phpcs:ignore WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___COOKIE
```

Comment:
The `WordPressVIPMinimum` ruleset is not part of this project, so it doesn't make sense to reference it. If this is causing problems for your install, I suggest ignoring it in a custom ruleset on your end.

```suggestion
	if ( isset( $_COOKIE['wordpress_sw_installed'] ) ) {
```